steeltoe-sample/src/solar-system-weather/Program.cs

```diff
@@ -37,10 +34,10 @@ public static void Main(string[] args)
         /// </summary>
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
+                .UseAzureSpringCloudService()
```

Comment:
Shall we add a comment here to explicitly tell user why we changed the sequence of `UseAzureSpringCloudService` ?